prototypes/Extension Point Prototype/.checkstyle

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<fileset-config file-format-version="1.2.0" simple-config="true" sync-formatter="false">
+  <local-check-config name="Beagle Checkstyle" location="D:\Meine Dateien\Uni\Semester B03 (WS1516)\PSE\Beagle\src\test\Checkstyle.xml" type="external" description="">
```

Comment:
Oh no, an absolute path! You want to use a workspace relative configuration instead of a local one here.